MCServer/Plugins/APIDump/APIDesc.lua

```diff
@@ -2491,6 +2503,7 @@ local CompressedString = cStringCompression.CompressStringGZIP("DataToCompress")
 				GetStorageSaveQueueLength = { Params = "", Return = "number", Notes = "Returns the number of chunks queued up for saving" },
 				GetTicksUntilWeatherChange = { Params = "", Return = "number", Notes = "Returns the number of ticks that will pass before the weather is changed" },
 				GetTimeOfDay = { Params = "", Return = "number", Notes = "Returns the number of ticks that have passed from the sunrise, 0 .. 24000." },
+				GetTNTShrapnelLevel = { Params = "", Return = "eShrapnelLevel", Notes = "Returns the level of Shrapnel." },
```

Comment:
A description like this is pretty useless - it provides no info for someone who has no idea what shrapnel level is, and it says nothing more than the function name itself.